package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
```

Comment:
I assume this was an accident but we don't want `package.json` or `package-lock.json` in the uppermost folder, these can be removed

Reply:
Removed